assets/css/WorkflowEditor/Report/_Block.scss

```diff
@@ -1,17 +1,32 @@
 article.report .report-container section.block {
+  padding: 1rem 2rem;
   display: flex;
-  margin: 1rem 2rem;
+  flex-direction: column;
 
   .block-main {
     flex: 1 1 auto;
-  }
 
+  }
   .block-actions {
     flex: 0 0 auto;
     display: flex;
-    flex-direction: column;
     justify-content: flex-end;
-    margin: 0 1rem 0 -1rem;
+    button {
+      font-size: 2rem;
```

Comment:
This font-size affects the `<IconButton>`, and that makes things blurry. Please nix this `font-size` so the `<IconButton>` can be 16px.